packages/diagram/src/module/diagrams/content/uml/classifier/providesAndRequires.ts

```diff
@@ -57,6 +69,15 @@ export const providesAndRequiresModule = InterpreterModule.create(
                                         )
                                         interfaceConnection.contents[0]._verticalPos = 1
                                         scope.internal.registerInDiagramScope(name, interfaceConnection)
+                                        (xLabelOffset, yLabelOffset) = args.namePos ?? [null, null]
+                                        nameLabelPos = canvasScope.rpos(interfaceConnection, xLabelOffset, yLabelOffset)
```

Comment:
yes
in particular here, as the two coordinates will be set by a style rule later if not defined here
but even if completely missing, they default to 0 (useful for when using graphical edits, so that you don't have to enter two numbers before being able to dragging the rpos around